advocacy_docs/partner_docs/QuestFoglightforDatabases/05-UsingQuestFoglightforDatabases.mdx

```diff
@@ -0,0 +1,74 @@
+---
+title: 'Using Quest Foglight for Databases to Monitor Health of EDB Postgres Advanced Server and/or EDB Postgres Extended Server Databases'
+description: 'Walkthroughs of Quest Foglight database monitoring'
+---
+This section will go through the menu items listed under a database in Quest Foglight to illustrate its monitoring capabilities.  After you have successfully created your EDB Postgres Advanced Server and/or EDB Postgres Extended Server agents to monitor, via the steps in the Configure Quest Foglight for EDB Postgres Advanced Server and EDB Postgres Extended Server, you will then have access to 24/7 monitoring and performance tracking via the Quest Foglight web browser.
```

Comment:
Change to "...you will then have 24/7 access to monitoring and performance tracking via the Quest Foglight web browser."